lightwood/encoders/text/short.py

```diff
@@ -0,0 +1,109 @@
+import torch
+from torch.nn.functional import pad
+from lightwood.encoders.categorical import CategoricalAutoEncoder
+
+
+def _get_tokens(text):
```

Comment:
In the future let's use a special tokenizer, for now let's add splitting at ` `, `,` `#`

Reply:
I made a temporaty tokenizer that will split by various punctuation-like symbols. If future, I guess we will use nltk for tokenization?

Reply:
If nltk is already required, yes, if not we should try to use Flair.

What was the issue with the flair tokenizers again ? I didn't quite get that during the call

Reply:
It has two tokenizers and both are weird (space tokenizer just cares about spaces, segtok tokenizer converts "hello,world!" to tok_1: "hello,world", tok_2: "!"

https://github.com/flairNLP/flair/blob/master/flair/data.py#L560